# Translation of OpenERP Server.
# This file contains the translation of the following modules:
#	* pdi_connector
#
msgid ""
msgstr ""
"Project-Id-Version: OpenERP Server 5.0.15\n"
"Report-Msgid-Bugs-To: support@openerp.com\n"
"POT-Creation-Date: 2010-12-01 06:12:13+0000\n"
"PO-Revision-Date: 2010-12-01 06:12:13+0000\n"
"Last-Translator: <>\n"
"Language-Team: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: \n"

#. module: pdi_connector
#: model:ir.actions.act_window,name:pdi_connector.act_open_pdi_instance_view
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_instance
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_instance_user
#: view:pdi.instance:0
msgid "PDI Instance"
msgstr ""

#. module: pdi_connector
#: constraint:ir.model:0
msgid "The Object name must start with x_ and not contain any special character !"
msgstr ""

#. module: pdi_connector
#: field:pdi.instance,trans_ids:0
msgid "Transformations"
msgstr ""

#. module: pdi_connector
#: view:pdi.task:0
#: field:pdi.task,param_ids:0
#: view:pdi.transformation:0
#: field:pdi.transformation,param_ids:0
msgid "Parameters"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(%s) Unknown error"
msgstr ""

#. module: pdi_connector
#: field:pdi.instance,note:0
msgid "Note"
msgstr ""

#. module: pdi_connector
#: field:pdi.task,instance_id:0
#: field:pdi.transformation,instance_id:0
msgid "Instance"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(7) The task couldn't be loaded from XML or the Repository"
msgstr ""

#. module: pdi_connector
#: model:ir.module.module,description:pdi_connector.module_meta_information
msgid "Module to manage Pentaho Data Integration\n"
"    - Start task or transformation define on the repository\n"
"    - Schedule a task\n"
"    "
msgstr ""

#. module: pdi_connector
#: constraint:ir.actions.act_window:0
msgid "Invalid model name in the action definition."
msgstr ""

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Basic"
msgstr ""

#. module: pdi_connector
#: view:pdi.instance:0
msgid "Repository information"
msgstr ""

#. module: pdi_connector
#: view:pdi.instance:0
#: field:pdi.trans.param,trans_id:0
#: view:pdi.transformation:0
msgid "Transformation"
msgstr ""

#. module: pdi_connector
#: field:pdi.task,state:0
#: field:pdi.transformation,state:0
msgid "Status"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(1) Errors occurred during processing"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(2) An unexpected error occurred during loading / running of the task"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(3) Unable to prepare and initialize this transformation"
msgstr ""

#. module: pdi_connector
#: field:pdi.task,level:0
#: field:pdi.transformation,level:0
msgid "Level"
msgstr ""

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v4.0"
msgstr ""

#. module: pdi_connector
#: help:pdi.instance,repo_pass:0
msgid "Enter the password for this repository"
msgstr ""

#. module: pdi_connector
#: selection:pdi.task,state:0
#: selection:pdi.transformation,state:0
msgid "Running"
msgstr ""

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_transformation
msgid "Transformation of a PDI instance"
msgstr ""

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Nothing"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(7) The transformation couldn't be loaded from XML or the Repository"
msgstr ""

#. module: pdi_connector
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_root
msgid "PDI Configuration"
msgstr ""

#. module: pdi_connector
#: field:pdi.instance,repo_pass:0
msgid "Password"
msgstr ""

#. module: pdi_connector
#: view:pdi.instance:0
msgid "Notes"
msgstr ""

#. module: pdi_connector
#: field:pdi.instance,repo_user:0
msgid "Username"
msgstr ""

#. module: pdi_connector
#: constraint:ir.ui.view:0
msgid "Invalid XML for View Architecture!"
msgstr ""

#. module: pdi_connector
#: view:pdi.instance:0
#: view:pdi.task:0
#: field:pdi.task.param,trans_id:0
msgid "Task"
msgstr ""

#. module: pdi_connector
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_userroot
msgid "PDI - ETL"
msgstr ""

#. module: pdi_connector
#: field:pdi.instance,name:0
#: field:pdi.instance,repo_name:0
#: field:pdi.task,name:0
#: field:pdi.task.param,name:0
#: field:pdi.trans.param,name:0
#: field:pdi.transformation,name:0
msgid "Name"
msgstr ""

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Rowlevel"
msgstr ""

#. module: pdi_connector
#: view:pdi.instance:0
#: view:pdi.task:0
#: view:pdi.transformation:0
msgid "Launch"
msgstr ""

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Debug"
msgstr ""

#. module: pdi_connector
#: help:pdi.instance,repo_user:0
msgid "Enter the username for this repository"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "pdi path does not exist"
msgstr ""

#. module: pdi_connector
#: field:pdi.task.param,value:0
#: field:pdi.trans.param,value:0
msgid "Value"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(9) Command line usage printing"
msgstr ""

#. module: pdi_connector
#: selection:pdi.task,state:0
#: selection:pdi.transformation,state:0
msgid "Stopped"
msgstr ""

#. module: pdi_connector
#: model:ir.module.module,shortdesc:pdi_connector.module_meta_information
msgid "PDI Connector"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
#, python-format
msgid "Error"
msgstr ""

#. module: pdi_connector
#: field:pdi.task,directory:0
#: field:pdi.transformation,directory:0
msgid "Directory"
msgstr ""

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_task
msgid "Task of a PDI instance"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(2) An unexpected error occurred during loading / running of the transformation"
msgstr ""

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Detailed"
msgstr ""

#. module: pdi_connector
#: view:pdi.instance:0
#: field:pdi.instance,task_ids:0
msgid "Tasks"
msgstr ""

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_instance
msgid "Instance of PDI server"
msgstr ""

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v3.2"
msgstr ""

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v4.1"
msgstr ""

#. module: pdi_connector
#: model:ir.actions.act_window,name:pdi_connector.act_open_pdi_transformation_view
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_transformation
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_transformation_user
msgid "PDI Transformation"
msgstr ""

#. module: pdi_connector
#: code:addons/pdi_connector/object/pdi.py:0
#, python-format
msgid "(8) Error loading steps or plugins (error in loading one of the plugins mostly)"
msgstr ""

#. module: pdi_connector
#: view:pdi.instance:0
msgid "General"
msgstr ""

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_task_param
msgid "Parameter to launch task"
msgstr ""

#. module: pdi_connector
#: field:pdi.instance,version:0
msgid "Version"
msgstr ""

#. module: pdi_connector
#: model:ir.actions.act_window,name:pdi_connector.act_open_pdi_task_view
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_task
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_task_user
msgid "PDI Task"
msgstr ""

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_trans_param
msgid "Parameter to launch transformation"
msgstr ""

#. module: pdi_connector
#: help:pdi.instance,repo_name:0
msgid "Enter the name of the repository"
msgstr ""

